Assets/HoloToolkit/Sharing/Scripts/SDK/DiscoveryClientAdapter.cs

```diff
@@ -2,6 +2,9 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```

Comment:
It doesn't, but the source files in the HoloToolkit repo still have these usings (and I used a script to copy everything over from there to the Unity project).  I can remove those there and then revert this part of the change in HoloToolkit-Unity.